spec/helpers/email_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "spec_helper"
+
+describe EmailHelper do
+  describe "#current_subdomain" do
+    it "returns empty subdomain if tenant is public" do
+      expect(helper.current_subdomain).to eq('')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.